GWAS/PCA.ipynb

```diff
@@ -233,112 +233,11 @@
     "kernel": "SoS"
    },
    "source": [
-    "## How to run on a MWE?\n",
```

Comment:
@haoyueshuai our other notebooks do have MWE commands and actually piopeline code in one place. That helps with portability. I suggest restore this section unless you have a compelling reason to do it in another separate notebook -- the link you provide points to a private repo that not everyone has access to.